client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/style.scss

```diff
@@ -23,24 +31,26 @@ $font-family: "SF Pro Text", $sans;
 	.step-container__content {
 		display: flex;
 		width: 100%;
-		margin: 0 32px;
 	}
 
 	.pattern-assembler__wrapper {
 		display: flex;
 		flex-direction: row;
 		flex: 1;
 		outline: none;
+		position: relative;
 	}
 
 	.pattern-assembler__sidebar {
 		display: flex;
 		flex-direction: column;
-		width: 284px;
+		width: 343px;
```

Comment:
This should be wrapped inside a class so it could be controlled by a feature flag.

Reply:
Thanks. I think we can make this change without the flag.